applications/GeoMechanicsApplication/custom_elements/Policies.md

```diff
@@ -0,0 +1,137 @@
+                                 \epsilon_{xy} \end{bmatrix}
+```
+Where $u_r$ is the radial displacement and $r$ is the radial coordinate. 
+
+### B-matrix
+The B matrix is used to relate strains and displacements. Therefore, its elements are filled with the spatial gradients of the shape functions ($N$).
+
+For a 3D element consisting of 4 nodes, the gradient looks like:
+```math
+\nabla{N} =
+\begin{bmatrix}
+\delta N_1/\delta x & \delta N_1/\delta y & \delta N_1/\delta z\\
+\delta N_2/\delta x & \delta N_2/\delta y & \delta N_2/\delta z\\
+\delta N_3/\delta x & \delta N_3/\delta y & \delta N_3/\delta z\\
+\delta N_4/\delta x & \delta N_4/\delta y & \delta N_4/\delta z
+\end{bmatrix}
+```
+Resulting in the following B-matrix:
+```math
+B =
+\begin{bmatrix}
+\delta N_1/\delta x & 0 & 0 & && & \delta N_4/\delta x & 0 & 0 \\
+0 & \delta N_1/\delta y & 0 & && & 0 & \delta N_4/\delta y & 0\\
+0 & 0 & \delta N_1/\delta z & &\dots& & 0 & 0 & \delta N_4/\delta z\\
+\delta N_1/\delta y & \delta N_1/\delta x & 0 & && & \delta N_4/\delta y & \delta N_4/\delta x & 0 \\
+0 & \delta N_1/\delta z & \delta N_1/\delta y & && & 0 & \delta N_4/\delta z & \delta N_4/\delta y \\
+\delta N_1/\delta z & 0 & \delta N_1/\delta x & && & \delta N_4/\delta z & 0 & \delta N_4/\delta x
+\end{bmatrix}
+```
+Where the first three columns are for the first node, the next three columns for the second node and so on for the number of nodes in the element.
+
+For a 2D element consisting of three nodes, the gradient looks like:
+```math
+\nabla{N} =
+\begin{bmatrix}
+\delta N_1/\delta x & \delta N_1/\delta y \\
+\delta N_2/\delta x & \delta N_2/\delta y \\
+\delta N_3/\delta x & \delta N_3/\delta y
+\end{bmatrix}
+```
+Where the rows depict the different nodes in the element (i.e. $N_i$ is the shape function for node $i$). For the plane strain case, this would result in the following B-matrix
+```math
+B =
+\begin{bmatrix}
+\delta N_1/\delta x & 0 & \delta N_2/\delta x & 0 & \delta N_3/\delta x & 0 \\
+0 & \delta N_1/\delta y & 0 & \delta N_2/\delta y & 0 & \delta N_3/\delta y\\
+0 & 0 & 0 & 0 & 0 & 0\\
+\delta N_1/\delta y & \delta N_1/\delta x & \delta N_2/\delta y & \delta N_2/\delta x & \delta N_3/\delta y & \delta N_3/\delta x
+\end{bmatrix}
+```
+For the axisymmetric stress state, this would result in the following B-matrix
+```math
+B =
+\begin{bmatrix}
+\delta N_1/\delta x & 0 & \delta N_2/\delta x & 0 & \delta N_3/\delta x & 0 \\
+0 & \delta N_1/\delta y & 0 & \delta N_2/\delta y & 0 & \delta N_3/\delta y\\
+N_1 / r & 0 & N_2 / r & 0 & N_3 / r & 0\\
+\delta N_1/\delta y & \delta N_1/\delta x & \delta N_2/\delta y & \delta N_2/\delta x & \delta N_3/\delta y & \delta N_3/\delta x
+\end{bmatrix}
+```
+
+### Integration coefficient
+
+For the 3D and plane strain stress states, the integration coefficient (or integration volume) is the product of the weight of the
+integration point and the determinant of the Jacobian:
+$$V_i = w_i \det{(J)}$$
```

Comment:
I guess here w_i is an area  (volume) for 2D (3D) case. Correct?

Reply:
yes, in the isoparametric coordinates. the det(J) takes it to the Cartesian coordinates
for the plane strain case the 3rd coordinate thickness is 1 by definition.
for axisymmetry it depends on the circumferential at is position.
for 3D no explanation needed.

Reply:
Thanks that's a good question, I'll try to make this part a bit more clear with this understanding. Please re-review!